cognite/powerops/custom_modules/power_model_v1/data_models/views/assets/PriceAreaDayAhead.view.yaml

```diff
@@ -25,16 +25,4 @@ implements:
     version: "{{version}}"
     type: view
 version: "{{version}}"
-properties:
-  defaultMethod:
-    container:
-      space: "{{powerops_models}}"
-      externalId: PriceArea
-      type: container
-    containerPropertyIdentifier: defaultMethodDayAhead
-    name: defaultMethod
-    source:
-      space: "{{powerops_models}}"
-      externalId: BidMethodDayAhead
-      version: "{{version}}"
-      type: view
+properties: {}
```

Comment:
What was the reason again for keeping this Dayahead extension?

Reply:
This price area is just for the front end right now, I _think_ and the front end things are still a TODO. It's at least not referenced in any of the compute models we're using for DayAhead